site/gatsby-site/src/pages/cite/edit.js

```diff
@@ -101,11 +102,27 @@ function EditCitePage(props) {
         },
       });
 
+      await updateIncident({
+        variables: {
+          query: {
+            incident_id: parentIncident.incident.incident_id,
+          },
+          set: {
+            reports: {
+              link: parentIncident.incident.reports
+                .filter((report) => report.report_number != reportNumber)
+                .map((report) => report.report_number),
+            },
+          },
+        },
+      });
+
       addToast({
         message: `Incident report ${reportNumber} deleted successfully.`,
         severity: SEVERITY.success,
       });
     } catch (e) {
+      console.error(e);
```

Comment:
leftover

Reply:
Perhaps this is a silly question, but is it bad to log an error when an actions fails? It seems like it could help some people report bugs.

Reply:
The error will get kind of lost there, we are using (or we should be using) Rollbar for this.

 I created this issue a long time ago, and I think this is where errors should be logged: https://github.com/responsible-ai-collaborative/aiid/issues/586

(Uncaught errors are already being captured and logged by Rollbar)

Reply:
Got it!